Classes/common/CDTIncrementalStore.m

```diff
@@ -16,6 +16,14 @@
 #import "CDTFieldIndexer.h"
 #import "CDTISGraphviz.h"
 
+@interface CDTISBatchUpdateResult : NSPersistentStoreResult
+@property (nonatomic, strong) id result;
+@property (nonatomic, assign) NSBatchUpdateRequestResultType resultType;
+@end
+
+@implementation CDTISBatchUpdateResult
```

Comment:
The implementation is required even if empty.  The link fails if I leave it out.